operations/research-and-development/product/development-process/jira-lifecycle.md

```diff
@@ -0,0 +1,35 @@
+# Jira ticket lifecycle
+
+In addition to Github Issues, we also use Jira tickets for managing chunks of work to be performed.
+
+## Ticket creation
+
+Follow these [guidelines](https://handbook.mattermost.com/operations/research-and-development/product/development-process/new-bug-tickets)
+
+## Ticket triage
+
+Ticket triage happens within the team's normal Sprint cycle. At that meeting, the team reviews all new tickets, assessing the scope, effort, and impact of the ticket. They make a decision of the priority of the ticket. Once its decided in triage meeting that the ticket should be worked on, the bug moves into its [first ticket state](#will-do).
+
+## Ticket states
```

Comment:
We could also add these states
 - Backlog
 - Select for Development (I think this is related to To Do? Maybe mention this in the description for the "To Do" state)
 - Done No PR (ticket is resolved without a pull request for various reasons, e.g. it couldn't be reproduced or it was deemed invalid. This could be combined with the "Done" state)
 - Close (QA has tested the ticket and closes it)